### ISSUE TITLE
Was the model  vit5-*-vietnews-summarization only trained with vietnews dataset?

### ISSUE THREAD

**Issue by MinhDang685** [vietai/ViT5#9]
Hi VietAI team, 

Thanks again for sharing your great work.
I can easily reproduce the ROUGE scores result (as stated in [your paper](https://arxiv.org/pdf/2205.06457.pdf)) on Vietnews dataset, with both **VietAI/vit5-base-vietnews-summarization** and **VietAI/vit5-large-vietnews-summarization**, by running the script provided [here](https://github.com/vietai/ViT5/blob/4248307081e7a99fbd35ce82c6fd70789ecc0014/eval/Eval_vietnews_sum.ipynb). But when I evaluate the 2 HuggingFace models, with Wikilingua (after change the tsv file to [wikilingua test set](https://raw.githubusercontent.com/vietai/ViT5/main/data/wikilingua/test.tsv ) to make it works for wikilingua)
I found that the ROUGE scores are noticeably worse than the ones in the paper
![image](https://user-images.githubusercontent.com/27952830/222346537-4567655e-d0fb-4005-9499-66c4ef3b038e.png)
I got ROUGE-1 = 50.01, ROUGE-2 = 17.28, ROUGE-L = 29.92 for the large model and ROUGE-1 = 48.83, ROUGE-2 = 16.26, ROUGE-L = 29.37 for the base model.

And with the name of the models "vit5-*-vietnews-summarization" I guess the 2 summarization models were only trained with vietnews dataset. Could you help to correct or confirm my assumption?

Thanks.
Minh

**Comment by justinphan3110** [comment]
@MinhDang685  our released huggingface model **VietAI/vit5-base-vietnews-summarization and VietAI/vit5-large-vietnews-summarization** are finetuned on vietnews dataset only. To reproduce the result on wikilingua you need to fine-tune it from the ViT5 checkpoints

**Comment by MinhDang685** [comment]
@justinphan3110 thank you for your reply
